src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -570,7 +605,7 @@ def load_status_file_components(self, initial_load=False):
 
         self.composite_logger.log_debug("Loading status file components [InitialLoad={0}].".format(str(initial_load)))
 
-        # Remove older complete status files
+        # Retain 10 complete status files, and remove older files
```

Comment:
Marking all changes not related to this PR goal. See my other comment on what can be done with these

Reply:
this kind of change is so small, it should be overlook. lets focus on the code logic and functionalities

Reply:
Ignore this for now to look at the bigger picture. Rajasi's guidance generally applies but I'm resolving to reduce noise.